karaf/init/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+        <artifactId>brooklyn-tosca-karaf</artifactId>
+        <version>0.10.0-SNAPSHOT</version> <!-- BROOKLYN_TOSCA_VERSION -->
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>brooklyn-tosca-karaf-init</artifactId>
+    <packaging>bundle</packaging>
+    <name>Brooklyn TOSCA Karaf OSGI Initialization</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+            <artifactId>brooklyn-tosca-karaf-deps-brooklyn</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+            <artifactId>_brooklyn-tosca-karaf-patches</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+            <artifactId>brooklyn-tosca-transformer</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+<!--
+                <version>3.2.0</version>
+-->
+                <extensions>true</extensions>
+                <configuration>
+                    <supportedProjectTypes>
+                        <!--
+                         Very strange that we have to include this section. The project is quite happy to build
+                         without specifying supportedProjectTypes if there is no 'src/main/resources' directory,
+                         but adding that directory causes the build to break with:
+                         [WARNING] Ignoring project type bundle - supportedProjectTypes = [jar]
```

Comment:
It's because of this [parent](https://github.com/apache/brooklyn-dist/blob/master/downstream-parent/pom.xml#L415). We should update brooklyn to use bundle packaging.